src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Magic number detected: `Color.values().length`. You should define a constant for the number of colors available, which will make the code more readable and maintainable.

Reply:
add public static final int maxOptionsColor = 6